pkg/rpctype/rpctype.go

```diff
@@ -37,6 +37,7 @@ type CheckArgs struct {
 	Leak           bool
 	Fault          bool
 	UserNamespaces bool
+	CompsEnabled   bool
```

Comment:
s/CompsEnabled/CompsSupported/
They are not enabled yet, they are supported by target.